e2e/e2etests/test_zevm_to_evm_call_revert_and_abort.go

```diff
@@ -0,0 +1,54 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/protocol-contracts/pkg/gatewayzevm.sol"
+
+	"github.com/zeta-chain/node/e2e/contracts/testabort"
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/testutil/sample"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+)
+
+func TestZEVMToEVMCallRevertAndAbort(r *runner.E2ERunner, args []string) {
+	require.Len(r, args, 0)
+
+	r.ApproveETHZRC20(r.GatewayZEVMAddr)
+
+	// deploy testabort contract
+	testAbortAddr, _, testAbort, err := testabort.DeployTestAbort(r.ZEVMAuth, r.ZEVMClient)
+	require.NoError(r, err)
+
+	// perform the withdraw
+	tx := r.ZEVMToEMVCall(
+		sample.EthAddress(), // non-existing address
+		[]byte("revert"),
+		gatewayzevm.RevertOptions{
+			RevertAddress:    sample.EthAddress(), // non-existing address
+			CallOnRevert:     true,
+			RevertMessage:    []byte("revert"),
+			OnRevertGasLimit: big.NewInt(200000),
+			AbortAddress:     testAbortAddr,
+		},
+	)
+
+	// wait for the cctx to be mined
+	cctx := utils.WaitCctxMinedByInboundHash(r.Ctx, tx.Hash().Hex(), r.CctxClient, r.Logger, r.CctxTimeout)
```

Comment:
nit: if you want directly use `cctx := utils.WaitCctxAbortedByInboundHash(r.Ctx, r, tx.Hash().Hex(), r.CctxClient)`.

Reply:
No, we should refactor or remove this one https://github.com/zeta-chain/node/issues/3544